src/components/explorer/SearchSummaryModal.js

```diff
@@ -52,7 +52,7 @@ const renderCharts = (chartsData) => {
 const SearchSummaryModal = ({ searchResults, ...props }) => {
     const [data, setData] = useState(null);
 
-    const katsuUrl = useSelector((state) => state.services.itemsByArtifact.metadata.url);
+    const katsuUrl = useService("metadata")?.url;
```

Comment:
Not related to this PR. The `katsuUrl` const is not in the useEffect's dependencies array, we probably want to check that `katsuUrl` is truthy before doing the fetch as well.

Reply:
same as above, good point